crates/swift-bridge-ir/src/codegen/codegen_tests/opaque_rust_type_codegen_tests.rs

```diff
@@ -81,6 +81,71 @@ void __swift_bridge__$SomeType$_free(void* self);
     }
 }
 
+/// Test code generation for an extern "Rust" type that implements Hashable.
+mod extern_rust_hashable_type {
+    use super::*;
+
+    fn bridge_module_tokens() -> TokenStream {
+        quote! {
+            mod ffi {
+                extern "Rust" {
+                    #[swift_bridge(Hashable)]
+                    type HashableType;
+                }
+            }
+        }
+    }
+
+    fn expected_rust_tokens() -> ExpectedRustTokens {
+        ExpectedRustTokens::Contains(quote! {
+        use std::hash::Hash;
```

Comment:
This is now failing. Just need to move it inside and then I can merge

Reply:
Sorry...After addressing your review, I forgot to run unit tests.